src/frontends/README.md

```diff
@@ -0,0 +1,15 @@
+# OpenVINO Frontends
+
+OpenVINO Frontends allow converting models from the native framework to OpenVINO representation.
```

Comment:
It would be nice to see this expanded a little bit (maybe in a future PR), and include Model Optimizer and OpenVINO Runtime using these frontends.

Reply:
Hi @helena-intel, agree I think we can put something like here: https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/index.md